app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/chart/RssiFilterLineChart.java

```diff
@@ -38,23 +35,18 @@ public RssiFilterLineChart(Context context, @Nullable AttributeSet attrs) {
     @Override
     public void initialize() {
         super.initialize();
+        filterList.add(new MeanFilter());
```

Comment:
Add `filterList = new ArrayList<>();` before this line in case `initialize` will be called multiple times. No need to initialize it in line 25.